app/aggregates/deciding/decision.rb

```diff
@@ -122,6 +122,7 @@ def send_to_provider(user_id:, application_id:)
     # Original reference is retained when linking to CIFC
     on LinkedToCifc do |event|
       @application_id = event.data.fetch(:application_id)
+      @state = Types::DecisionState[:draft]
```

Comment:
☝️ reset the state to draft when a decision is linked to CIFC. (This was the source of the bug preventing removal.) (Non CIFC applications cannot be linked to a previously linked application.)